SwrveSDK/SDK/Talk/SwrveMessage.m

```diff
@@ -68,12 +41,28 @@ -(SwrveMessage*)updateWithJSON:(NSDictionary*)json
     self.formats = [[NSArray alloc] initWithArray:loadedFormats];
     return self;
 }
+-(SwrveMessageFormat*)getBestFormatFor:(UIInterfaceOrientation)orientation
+{    
```

Comment:
~~ [super predantic here] but the format for functions are the same as Java in this case:
function and `{` are one space apart and on the same line. 
` - (object *) function:(argument) {

}`